contracts/vendor/starkware-libs/starkgate-contracts/src/starkware/solidity/tokens/ERC20/ERC20.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: Apache-2.0.
```

Comment:
Check if these contracts can be imported from OpenZeppelin via npm instead of copying? Check if there are any differences

Reply:
Yeah, we can use the OZ Solidity ERC20 implementation by simply importing their NPM package. There is no need to use the StarkGate ERC20 or copy the contract over.

Reply:
Moved